plugin/keyring_vault/tests/mtr/innodb_online_alter_encryption.test

```diff
@@ -1,6 +1,7 @@
 --source include/have_keyring_vault_plugin.inc
+--source include/not_repeat.inc
```

Comment:
Did you try to add `--force-restart` to the `.opt` file instead of this `not_repeat.inc`?